files/usr/sbin/jeos-firstboot

```diff
@@ -4,14 +4,22 @@
 
 set -e
 
+# set to --keep-tite to use altenate screen. Better for debugging but causes flicker
+dialog_altenate_screen=
```

Comment:
Typo. This is used in jeos-firstboot-functions and needs to be defined there. Currently it's undefined when running jeos-config.